requirements/check-pins.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#
+# Compare two pip pin files.  Everything in the first one should match one
+# in the second one.
+
+import re
+import sys
+
+def pins(lines):
+    for line in lines:
```

Comment:
pylint probably isn't running on this file, may want to update its arguments in `tox.ini` to fix that.

Reply:
I've added pylint.